src/pybind/mgr/dashboard/frontend/src/app/shared/components/copy2clipboard-button/copy2clipboard-button.component.html

```diff
@@ -0,0 +1,9 @@
+&nbsp;
+<button (click)="onClick()"
+        type="button"
+        class="btn btn-light"
+        i18n-title
+        title="Copy to Clipboard">
+<i class="fa fa-clipboard">
```

Comment:
You can make this as
```suggestion
<i [ngClass]="[icons.clipboard]"></i>
```
and also import the `icons.enum.ts` in the `copy2clipboard-button.component.ts `